### PR TITLE
pulseaudio: fix input mute toggle

### LINKED ISSUE #14: swayosd --input-volume mute-toggle mutes output
When I execute `swayosd --input-volume mute-toggle` to toggle my microphone, I get the regular "microphone volume" popup (volume at 100%), but instead of the microphone being toggled, the output volume gets muted. I'm using swayosd 5c2176a, installed from Arch AUR. Running `swayosd --input-volume mute-toggle` again does nothing (so it does not unmute the output volume)

Steps to reproduce:
1. Make sure input and output volume are unmuted (e.g. via pavucontrol)
2. Execute `swayosd --input-volume mute-toggle`
3. Observe 'mic 100%' popup
4. Observe output volume is muted

### PR DESCRIPTION
**Pull request by mochaaP**
close #14

### CODE REVIEW

**Comment by mochaaP** [comment]
close #14

File: src/utils.rs [hunk 1]
```diff
@@ -41,9 +41,11 @@ pub fn volume_parser(is_sink: bool, value: &str) -> Result<(ArgTypes, Option<Str
 	if is_sink {
 		if v.0 == ArgTypes::SinkVolumeRaise {
 			v.0 = ArgTypes::SourceVolumeRaise;
-		} else {
+		} else if v.0 == ArgTypes::SinkVolumeLower {
 			v.0 = ArgTypes::SourceVolumeLower;
-		}
+		} else {
+            v.0 = ArgTypes::SourceVolumeMuteToggle;
+        }
```
Comment (ErikReider) [inline src/utils.rs:48]: Use tabs instead of spaces

**Review by ErikReider — CHANGES_REQUESTED** [review]
Thanks for the help! :) Just one minor nit

File: src/utils.rs (hunk 1 above)
Comment (mochaaP) [reply src/utils.rs]: ack

**Review by ErikReider — APPROVED** [review]
LGTM! Thanks again! :)

**Pull request merged**